SG.js

```diff
@@ -529,3 +561,8 @@ loadRegdateCache();
 function saveRegdateCache() {
 	fs.writeFileSync('config/regdate.json', JSON.stringify(regdateCache));
 }
+
+function getEXPType(pokemon) {
+	let data = JSON.parse(fs.readFileSync('config/SGGame-Data/EXP.json', 'utf8'));
+	return data[pokemon].expType;
```

Comment:
Not sure how wise it is to sync read from json everytime we want to get the EXP type of a pokemon.